src/ceph.in

```diff
@@ -515,6 +517,42 @@ else:
                 return line
 
 
+def do_command(parsed_args, target, cmdargs, sigdict, inbuf, verbose):
+    ''' Validate a command, and handle the polling flag '''
+
+    valid_dict = validate_command(sigdict, cmdargs, verbose)
+    # Validate input args against list of sigs
+    if valid_dict:
+        if parsed_args.output_format:
+            valid_dict['format'] = parsed_args.output_format
+
+        if verbose:
+            print("Submitting command: ", valid_dict, file=sys.stderr)
+    else:
+        return -errno.EINVAL, '', 'invalid command'
+
+    if 'poll' in valid_dict and valid_dict['poll']:
+        while True:
+            # Run the command and print the result
+            ret, outbuf, outs = json_command(cluster_handle, target=target, argdict=valid_dict,
+                inbuf=inbuf)
+
+            if ret:
+                ret = abs(ret)
+                print('Error: {0} {1}'.format(ret, errno.errorcode.get(ret, 'Unknown')),
+                        file=sys.stderr)
+            if outbuf:
+                print(outbuf)
+            if outs:
+                print(outs, file=sys.stderr)
+            sleep(1)
```

Comment:
Would be nice if the polling interval was configurable, defaulting to some value (may it be 1 or something else). Maybe an option passed to the cli tool?

Reply:
Right, I was thinking of having it in the command description itself, but I think a flag to the cli tool is just as valid and definitely easier. Done.